index.cjs

```diff
@@ -0,0 +1,6 @@
+// This is just a CommonJS wrapper for the default export to not to introduce a
+// breaking change on how WebBundlePlugin is imported.
+// For context, see:
+// https://github.com/evanw/esbuild/issues/532#issuecomment-1019392638
```

Comment:
I have thought about this for a bit now, and I think I agree with what you did - IMHO we should breaking-change the export from default to non-default when we eventually release 1.0.0 of this package.

Reply:
Sounds good! I'll add a TODO there.